src/libraries/System.Net.Http.Json/src/System.Net.Http.Json.csproj

```diff
@@ -0,0 +1,20 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="System\Net\Http\Json\HttpClientJsonExtensions.Get.cs" />
+    <Compile Include="System\Net\Http\Json\HttpClientJsonExtensions.Post.cs" />
+    <Compile Include="System\Net\Http\Json\HttpClientJsonExtensions.Put.cs" />
+    <Compile Include="System\Net\Http\Json\HttpContentJsonExtensions.cs" />
+    <Compile Include="System\Net\Http\Json\JsonContent.cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <Reference Include="System.Runtime" />
+    <Reference Include="System.Net.Http" />
+    <Reference Include="System.Net.Primitives" />
+    <Reference Include="System.Text.Json" />
```

Comment:
What version of the System.Text.Json package does this result in?

Reply:
Here in the runtime repo, it will result in 5.0.0-preview version. Once this is ported over to release/3.1 repo, then it will mean reference whatever latest stable package is available in NuGet.org

Reply:
>  whatever latest stable package is available in NuGet.org

Yes, effectively.  And if we happen to be sim-shipping a fix to S.T.J in servicing and decide that we should use that one for all references, it can be that too.